### DISCUSSION TITLE
Could this be made to work in Visual Studio?

### DISCUSSION THREAD

**Discussion by Cafetorium** [prb28/vscode-amiga-assembly#289] [General]
Having used Visual Studio for many years, I find that Visual Studio Code is a severe downgrade.  I don't particularly want to use it, and definitely don't want both VS and VSCode installed on my PC.
Is it possible to make this into a plugin for Visual Studio?

**Comment by prb28** [comment]
I don't have the time to make this possible but maybe someone will be interested.
